invenio_assets/ext.py

```diff
@@ -59,7 +59,7 @@ def init_config(self, app):
         )
 
         # Flask-WebpackExt config
-        app.config.setdefault("WEBPACKEXT_PROJECT", "invenio_assets.webpack:project")
+        app.config.setdefault("WEBPACKEXT_PROJECT", "invenio_assets.webpack:webpack_project")
```

Comment:
the original value is still provided for backwards compatibility, but moving forward i'd suggest going with the new fallback value